homeassistant/components/sensor/london_underground.py

```diff
@@ -119,15 +119,21 @@ def parse_api_response(response):
     data_dict = dict.fromkeys(lines)
 
     for line in response:
+
         statuses = [status['statusSeverityDescription']
                     for status in line['lineStatuses']]
         state = ' + '.join(sorted(set(statuses)))
 
         if state == 'Good Service':
             reason = 'Nothing to report'
         else:
-            reason = ' *** '.join(
-                [status['reason'] for status in line['lineStatuses']])
+            reason = " *** ".join(
```

Comment:
The service interface, I/O and data parsing, should be moved to a standalone library published on pypi.

https://developers.home-assistant.io/docs/en/creating_platform_code_review.html#6-communication-with-devices-services

Reply:
Will do